dashboard/app/models/follower.rb

```diff
@@ -53,4 +53,15 @@ def student_cannot_be_admin
   def assign_script
     student_user.assign_script(section.script) if section.script
   end
+
+  after_destroy :remove_family_name, if: proc {DCDO.get('family-name-features', false)}
```

Comment:
I understand that the feature is behind a DCDO flag but should this hook be behind it? If we turned on `family-name-features` then turned it off again, we'd still want this hook to run right?

Reply:
I'm definitely open to that. My thinking here was that, in the off chance this change does end up breaking something, we'd have a quick off-switch without having to redeploy. And having some extra data in user properties that we could manually clean up if necessary, would be a smaller issue than breaking the ability to remove users from sections.

Reply:
Fair point -- I'm okay with that plan. Just wanted to make sure we'd thought of this scenario!